Makefile

```diff
@@ -103,7 +103,7 @@ $(BINDIR) $(BINDIR)/chart:
 	mkdir -p $@
 
 $(BINDIR)/ginkgo:
-	GOBIN=$(BINDIR) go install github.com/onsi/ginkgo/ginkgo
+	GOBIN=$(BINDIR) go install github.com/onsi/ginkgo/v2/ginkgo
```

Comment:
comment: it hurts my soul to see this Makefile, there are so many things wrong with it 😂  Obviously not a blocker for this PR, but my god...